sf-docker/.env.sfac

```diff
@@ -0,0 +1 @@
+#FilterConfiguration__SpoofCheckConfiguration__Enabled=true
```

Comment:
do we really want this enabled by default?

Reply:
it is commented - just a sample how it should be composed